### ISSUE TITLE
Broken on React-native 0.60 because of ViewPagerAndroid

### ISSUE THREAD

**Issue by Fortidude** [react-native-community/react-native-viewpager#47]
# Bug

Application crashes on when trying to use this library. The output error:

```JS
ExceptionsManager.js:86 Invariant Violation: Invariant Violation: Unable to find node on an unmounted component.

This error is located at:
    in ViewPagerAndroid
```
I can see it's located in ViewPagerAndroid, but according to React-Native documentation, from ViewPageAndroid it says: 

>     Deprecated. Use react-native-community/react-native-viewpager instead.

So here I am.

## Environment info

**react: 16.8.6 => 16.8.6**
**react-native: 0.60.4 => 0.60.4**

**Library version: 1.1.7**
_why? In the release list I can't see it_

## Steps To Reproduce

1. yarn add @react-native-community/viewpager
2. react-native link (actually I tried without link as well, since it's RN 0.60)
3. copy-paste the code from "Usage" section in the main Readme of this library
4. run the app by react-native run-android
5. Crash

Describe what you expected to happen:

1. The application should start normally and display the demo swiper

## Reproducible sample code

<!--

**Comment by pmyagkov** [comment]
This is actually a showstopper. We can't deploy a new version of the app because of the issue. After painful upgrade to 0.60.*.

**Comment by ferrannp** [comment]
Hey,

I just created a new project with `react-native init` which contains `react-native@0.60.5`, I ran `yarn add @react-native-community/viewpager` and then `react-native run-android` with the README example and everything works fine.

Notice that `react-native run-android` will run `jetifier`:

```
❯ react-native run-android
info Running jetifier to migrate libraries to AndroidX. You can disable it using "--no-jetifier" flag.
Jetifier found 869 file(s) to forward-jetify. Using 4 workers...
```

With this it should work. I gotta make a new version of the library that is compatible with Android X by default but in the meantime, jetifier should do the trick.
